mainflux/edge
# UV-278 - Modbus Adapter

### What does this do?
Creates Modbus adapter on mainflux edge

### Which issue(s) does this PR fix/relate to?
https://github.com/ultravioletrs/issues/issues/278

### List any changes that modify/break current functionality

### Have you included tests for your changes?

### Did you document any new/modified functionality?

### Notes

To test using a simulated device, 
```yaml
version: "2.1"
services:
  modbus:
    image: techplex/modbus-sim:latest
    container_name: modbus
    ports:
      - 1502:1502
    restart: unless-stopped
```
See readme for examples
